### ISSUE TITLE
Default slide width is different for dev build and production

### ISSUE THREAD

**Issue by tlakomy** [FormidableLabs/spectacle#352]
I've started working on some slides using Spectacle and I've noticed that when using the default example the default width of slide's content was different than on the demo on Spectacle webpage:

![screen shot 2017-08-27 at 17 53 17](https://user-images.githubusercontent.com/16646517/29751925-06c6cc88-8b55-11e7-8225-53788606cc34.png)

(FYI : I'm using current version Chrome 60.0.3112.113 on Mac.)

I'd like to be able to change the default width of all slides at once, AFAIK the current way to do that is to add `maxWidth={value}` to the `<Slide>` component, which can be a bit DRY when working on more than 5 slides.

**Comment by tomaszlakomygp** [comment]
Okay, so the issue mentioned in my previous comment is a bit different. It turns out that there's a difference in slide width between dev build and a production build, see below:

![screen shot 2017-08-27 at 19 57 50](https://user-images.githubusercontent.com/16646517/29752621-0ad66c0e-8b62-11e7-8eae-c67bc7f9e6b4.png)

**Comment by tomaszlakomygp** [comment]
I've debugged this issue a bit because I had a free evening and it seems to be caused by differences in `slide.js` `setZoom` function between development and production builds. If you put a debugger inside of this function, you'll get different values for `offsetWidth`/`offsetHeight` depending on which build you are debugging.

Development build:
![screen shot 2017-08-27 at 23 16 26](https://user-images.githubusercontent.com/16646517/29754137-6ad66498-8b7f-11e7-8c1e-65424939ebf0.png)


Production build:
![screen shot 2017-08-27 at 23 16 34](https://user-images.githubusercontent.com/16646517/29754138-6ecfcab2-8b7f-11e7-86c2-fda47cad638a.png)

I've used a following simple deck to minimise potential error factors:

```js
export default class Presentation extends React.Component {
  render() {
    return (
      <Deck transition={["zoom", "slide"]} theme={theme} transitionDuration={500}>
        <Slide transition={["zoom"]} bgColor="primary">
          <Heading size={1} caps lineHeight={1} textColor="black">
            Spectacle
          </Heading>
        </Slide>
      </Deck>
    );
  }
}
```

Hope this will help a bit.
